lib/src/components/NxToast/NxToast.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState, useContext, useEffect } from 'react';
+import classnames from 'classnames';
+import FocusContext from './contexts';
+
+import { nxToastPropTypes, NxToastProps, ToastModel } from './types';
+
+export { ToastModel };
+
+const NxToast = (props: NxToastProps) => {
+  const { onClose, children } = props;
+
+  const [isClosing, setIsClosing] = useState(false);
+
+  const focusedToast = useContext(FocusContext);
```

Comment:
Why is this called `focusedToast`?

Reply:
> Why is this called `focusedToast`?

Renamed here: [f3762d4](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/f3762d4c47a3fff4dd9a33336d29f28734101526)